presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkInjectorFactory.java

```diff
@@ -131,6 +132,7 @@ public Injector create()
         ImmutableList.Builder<Module> modules = ImmutableList.builder();
         modules.add(
                 new JsonModule(),
+                new SmileModule(),
```

Comment:
would it be possible to put this in `additionalModules` and bind when config is enabled?

Reply:
It seems like there is no existing interface to allow the change,  since this would be always enabled (unless there is regression), it should be fine. No worry.